package.json

```diff
@@ -52,19 +52,21 @@
     ]
   },
   "dependencies": {
+    "@babel/compat-data": "7.8.0",
```

Comment:
Where is it used?

Reply:
was fixing babel problem from match 20, trying to run the 5.6 version, remove this dep after next commit